app/app_client_test.go

```diff
@@ -989,6 +989,19 @@ func TestAppClient(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 	})
 
+	t.Run("EnableBillingSevice", func(t *testing.T) {
```

Comment:
do we need code in app_client?  whats this testing then?

Reply:
```
// EnableBillingService enables a billing service to an address in an organization.
func (c *AppClient) EnableBillingService(ctx context.Context, orgID string, billingAddress *BillingAddress) error {
	_, err := c.client.EnableBillingService(ctx, &pb.EnableBillingServiceRequest{
		OrgId:          orgID,
		BillingAddress: billingAddressToProto(billingAddress),
	})
	return err
}
```
this is already in the code ^